tests/base.py

```diff
@@ -197,6 +197,8 @@ def admin_token_assert_equal(self, query, expected_response):
         response = self.app_test.post(
             '/mrm?query=' + query, headers=headers)
         actual_response = json.loads(response.data)
+        print('query>>>>>>>>>>', query)
```

Comment:
Excellent work Obinna. I wanted to know if keeping the print statements on line 200 and 201 was necessary.